README.md

```diff
@@ -1,97 +1,46 @@
-[![Build Status](https://app.bitrise.io/app/e783d140de7df9d9/status.svg?token=RsL0h68Nd4D8wA_CtODchQ&branch=master)](https://app.bitrise.io/app/e783d140de7df9d9)
-
-# Go list
-
-Go list
-
-## How to use this Step
-
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
-
-_Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!_
-
-Step by step:
-
-1. Open up your Terminal / Command Line
-2. `git clone` the repository
-3. `cd` into the directory of the step (the one you just `git clone`d)
-4. Create a `.bitrise.secrets.yml` file in the same directory of `bitrise.yml`
-   (the `.bitrise.secrets.yml` is a git ignored file, you can store your secrets in it)
-5. Check the `bitrise.yml` file for any secret you should set in `.bitrise.secrets.yml`
-
-- Best practice is to mark these options with something like `# define these in your .bitrise.secrets.yml`, in the `app:envs` section.
-
-6. Once you have all the required secret parameters in your `.bitrise.secrets.yml` you can just run this step with the [bitrise CLI](https://github.com/bitrise-io/bitrise): `bitrise run test`
-
-An example `.bitrise.secrets.yml` file:
-
-```
-envs:
-- A_SECRET_PARAM_ONE: the value for secret one
-- A_SECRET_PARAM_TWO: the value for secret two
+# Go list [![Bitrise Build Status](https://app.bitrise.io/app/e783d140de7df9d9/status.svg?token=RsL0h68Nd4D8wA_CtODchQ&branch=master)](https://app.bitrise.io/app/e783d140de7df9d9) [![Bitrise Step Version](https://img.shields.io/badge/version-0.10.1-blue)](https://www.bitrise.io/integrations/steps/go-list) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/bitrise-steplib/steps-go-list/master/LICENSE) [![Bitrise Community](https://img.shields.io/badge/community-Bitrise%20Discuss-lightgrey)](https://discuss.bitrise.io/)
+
+This step runs the `go list ./...` command for you to list the go packages named by the import paths, starting from the current working directory.  
+It can return a filtered package list in line with the exclude patterns.
+
+## Examples
+
+### List packages in the working directory excluding vendor/*
```

Comment:
How can we auto-generate this Examples section for the steps' readmes?

Reply:
not sure if we can, or we should. 🤔

It's going to be either a manual process, to help the customers to understand it better, or we can use the bitrise.yml file to populate some examples from there